website/docs/language/state/purpose.mdx

```diff
@@ -22,9 +22,9 @@ shifting massive amounts of complexity from one place (state) to another place
 ## Mapping to the Real World
 
 Terraform requires some sort of database to map Terraform config to the real
-world. When you have a resource `resource "aws_instance" "foo"` in your
-configuration, Terraform uses this map to know that instance `i-abcd1234`
-is represented by that resource.
+world. For example, when you have a resource `resource "aws_instance" "foo"` in your
+configuration, Terraform uses this mapping to know that the resource `resource "aws_instance" "foo"`
+represents a real world object having instance ID `i-abcd1234` on a remote system.
```

Comment:
```suggestion
world. For example, when you have a resource `resource "aws_instance" "foo"` in your
configuration, Terraform uses this mapping to know that the resource `resource "aws_instance" "foo"`
represents a real world object with the instance ID `i-abcd1234` on a remote system.
```

Reply:
@laurapacilio - "with the" looks more appropriate. Thank you.